packages/vite-plugin-nitro/src/lib/vite-nitro-plugin.spec.data.ts

```diff
@@ -0,0 +1,51 @@
+import { NitroConfig } from 'nitropack';
```

Comment:
I have created this file so we don't pollute the test file with these helper functions and mock data. Of course I can put it back if you want.